docs/bsp.md

```diff
@@ -1308,6 +1313,23 @@ object ScalaPlatform {
 }
 ```
 
+#### Scala Test Params
+
+`ScalaTestParams` is a basic data structure that contains scala-specific
+metadata for testing Scala targets. This metadata is embedded in the `data:
+Option[Json]` field of the `buildTarget/test` request.
+
+```scala
+trait ScalaTestParams {
+  /** The test classes to be run in this test execution.
+    * It is the result of `buildTarget/scalaTestClasses`. */
+  def testClasses: List[ScalaTestClassesItem]
+
+  /** Additional client-specific data that the server can parse and understand. */
+  def data: Option[Json]
```

Comment:
Can we remove this field? If the client wants to send additional data then it can use `TestParams.data`, there is no need for double `.data.data` nesting.

Reply:
Addressed in latest commit.